File: Dataset/Leetcode/valid/11/39.java
class Solution {
    public int XXX(int[] height) {
        if(height == null || height.length == 0){
            return 0;
        }
        int i = 0;
        int j = height.length-1;
        int area = 0;
        int XXX = 0;
        while(i < j){
            if(height[i] > height[j]){
                area = height[j]*(j-i);
                j--;
            }else if(height[i] < height[j]){
                area = height[i]*(j-i);
                i++;
            }else{
                area = height[i]*(j-i);
                i++;
                j--;
            }
            if(XXX < area)
                XXX = area;
        }
        return XXX;
    }
}

File: Dataset/Leetcode/train/70/39.cpp
class Solution {
public:
    int XXX(int n) {
        if(n == 1 || n == 2) return n;
        int f[n+1];
        f[1] = 1;
        f[2] = 2;
        for(int i=3;i<=n;i++) f[i] = f[i-1] + f[i-2];
        return f[n];
    }
};

